omod/src/main/webapp/fragments/patientsearch/patientSearchWidget.gsp

```diff
@@ -36,9 +35,18 @@
     <%      }
         }%>
     function handlePatientRowSelection() {
+    	var afterSelectedUrl = '${ ui.escapeJs(config.afterSelectedUrl) }';
         this.handle = function (row) {
             var uuid = row.uuid;
-            location.href = '/' + OPENMRS_CONTEXT_PATH + emr.applyContextModel('${ ui.escapeJs(config.afterSelectedUrl) }', { patientId: uuid, breadcrumbOverride: '${ ui.escapeJs(breadcrumbOverride) }'});
+            if(afterSelectedUrl && afterSelectedUrl != 'null') {
+            	location.href = '/' + OPENMRS_CONTEXT_PATH + emr.applyContextModel(afterSelectedUrl, { patientId: uuid, breadcrumbOverride: '${ ui.escapeJs(breadcrumbOverride) }'});
```

Comment:
Did you intentionally change the above line from what it used to look like on line 41?

Reply:
yeah, since i had pulled out afterSelectedUrl into a variable